packaging/dbscripts/storages_san_sp.sql

```diff
@@ -542,6 +542,36 @@ BEGIN
 END;$PROCEDURE$
 LANGUAGE plpgsql;
 
+CREATE OR REPLACE FUNCTION GetStorageServerConnectionsByConnectionPortAndIqn (
+    v_iqn VARCHAR(128),
+    v_connection VARCHAR(250),
+    v_port VARCHAR(50)
+    )
+RETURNS SETOF storage_server_connections STABLE AS $PROCEDURE$
+BEGIN
+    RETURN QUERY
+
+    SELECT *
+    FROM storage_server_connections
+    WHERE (
+            iqn = v_iqn
+            OR (
+                iqn IS NULL
+                AND v_iqn IS NULL
+                )
+            )
+        AND (connection = v_connection)
+        AND (
+            port = v_port
+            OR (
```

Comment:
If `port` and `v_port` are `NULL`, wouldn't the first part `port = v_port` catch it?
Is this `OR` required?

Reply:
Good point. I actually copied it from the previously used query, but I can't quite understand what is the purpose of this check. In our case these values should not be null anyway.